.github/workflows/build-and-run-example.yml

```diff
@@ -1,4 +1,4 @@
-name: Build and run example
+name: Run random example
```

Comment:
We'd want to keep the "Build" here, since the main difference between this workflow and the changed example workflow is that it also builds the image by ignoring cached images. We use this to track regressions in builds.